API.md

```diff
@@ -120,3 +120,16 @@ Calls `sendBytes()` with `{ 0x00 }` as payload to poll for incoming messages.
 Returns the result of `sendBytes()`.
 
 See the [Receive](https://github.com/TheThingsNetwork/arduino-device-lib/blob/master/examples/Receive/Receive.ino) example.
+
+## Method: provision
+Sets the information needed to activate the device.
+
+```c
+bool provision(const byte appEui[8], const appKey[16]);
+```
+
+- `const byte appEui[8]`: Application Identifier for the device.
+- `const byte appKey[16]`: Application Key assigned to the device.
+
+Set the AppKey and AppEui in the beginning of the exemples.
+see the [Receive](https://github.com/TheThingsNetwork/arduino-device-lib/blob/master/examples/Receive/Receive.ino) exemple.
```

Comment:
The Receive examples doesn't have and will not have a `provision()`, so just leave this out.